README.md

```diff
@@ -514,7 +514,7 @@ Thus when you create a TODO, it is almost always your name that is given.
      end
      ```
 
-* <a name="no-default-args"></a>Do not use default arguments. Use an options
+* <a name="no-default-args"></a>Do not use default positional arguments. Use named arguments or an options
```

Comment:
FWIW, per that link, the performance hit is comparable to using `options` hash (since the issues it cites are around the implied construction/consumption of a `Hash`). If `options` hash is indeed a blessed pattern from a readability/extensibility > performance perspective, should we just recommend keyword arguments from the moment of availability?